store/inmem/InMem.go

```diff
@@ -82,11 +98,14 @@ func (i *InMem) GetAll(bucket string) (map[string]store.OwnableItem, error) {
 	i.lock.Lock()
 	defer i.lock.Unlock()
 	items := i.data[bucket]
-	if items == nil {
-		items = map[string]expireableItem{}
+	result := make(map[string]store.OwnableItem)
+	for idx := range items {
+		item := items[idx]
+		if expired := i.updateTTL(&item, items, bucket, item.ID); !expired {
```

Comment:
We don't need this variable 🥳 
```suggestion
		if !i.updateTTL(&item, items, bucket, item.ID) {
```
I do see the merit of keeping it in to have self-documenting code...people will understand why this happens when `updateTTL` returns a `false` value (the item is expired).  I've thought about maybe renaming the `updateTTL` function? but don't have a good suggestion currently. 😅